stock_scanner/README.rst

```diff
@@ -0,0 +1,241 @@
+These scenarios, are automatically imported when installing a new database with demo data.
+
+Installation
+============
+
+
+The "sentinel.py" specific ncurses client is available in the "hardware" directory.
+This application is a separate client, and can be run on any device.
+
+For mobile devices, like Windows Mobile or Android smart barcode scanners, we usually install it on a server, accessed through SSH.
+
+If you plan to use the specific "sentinel.py", you will need the "openobject-library" Python module, available from pip:
+
+    $ sudo pip install "openobject-library<2"
+
+.. note::
+
+   You must use openobject-library earlier than 2.0 with Odoo.
+   The version 2.0 of openobject-library only implements the Net-RPC protocol, which was removed from v7.
+
+To test the module, some modules provide scenario.
+
+Configuration
+=============
+
+In Odoo
+-------
+
+Declare hardware
+^^^^^^^^^^^^^^^^
+
+You have to declare some hardware scanners in Odoo.
+
+Go to "Warehouse > Configuration > Scanner Hardware" and create a new record.
+
+The "step type code" sent by the "sentinel.py" client at start-up is the IP address of the hardware, if connected through SSH.
+
+If needed enable Login/Logout
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The module come with 2 predifined scenarii for Login and Logout. The functionality is disabled by default and the user to use in
+Odoo must be specified in the .oerp_sentinelrc file used by sentinel and can be overriden on the Scanner Hardware definition
+in Odoo. 
+
+If the Login/logout functionality is enabled, when a user start a session with sentinel, only the Login scenario is displayed on the
+screen. The scenario will prompt the user for its login and pwd. If the authentication succeed, each interaction with Odoo will be done
+using the uid of the connected user. Once connected, a Logout scenario is displayed in the list of available scenarii and the Login
+scenario no more appear. 
+
+The Login/logout functionality enable you to specify on the scenario a list of users and/or a list of groups with access to the scenario.
+
+To enable the Login/logout functionality:
+    * Go to "Settings > Warehouse" and check the checkbox Login/logout scenarii enabled.
+    * Create a *Technical User* 'sentinel' **without roles in Human Resources** and with 'Sentinel: technical users' checked.
+    * Use this user to launch your sentinel session.
+
+Be careful, the role *Sentinel: technical users* is a technical role and should only be used by sentinel.
+
+The timeout of sessions is managed by a dedicated cron that reset the inactive sessions. The timeout can be configured on 
+settings. "Settings > Warehouse"
+
+For the sentinel.py client
+--------------------------
+
+The sentinel.py client uses a config file in the standard `ini` format.
+This file can be named `.oerp_sentinelrc`, `.openerp_sentinelrc` or `.odoo_sentinelrc`, and can be located in the current working directory, or in the user's home directory.
```

Comment:
I think that could be nice to add a .odoo_sentinelrc.sample file, and add stock_scanner/hardware/.odoo_sentinelrc to gitignore, it would be usefull to test enviroment.

Also it's not clear that this config file is not created atomatically and must to be created manually.

I copy sentinel.py on own directory, create .odoo_sentinelrc, and raises exception, you must also copy i18n folder, and it's not documented